pkg/service/test/test.go

```diff
@@ -707,21 +713,22 @@ func (t *tester) RunTestSet(testSet, path, testReportPath, appCmd, appContainer,
 		t.logger.Warn("These testcases have not been recorded by Keploy, may not work properly with Keploy.", zap.Strings("non-keploy mocks:", nonKeployTcs))
 	}
 	resultsCfg := &FetchTestResultsConfig{
-		TestReportFS:   initialisedValues.TestReportFS,
-		TestReport:     initialisedTestSets.TestReport,
-		Status:         &status,
-		TestSet:        testSet,
-		Success:        &success,
-		Failure:        &failure,
-		Ctx:            initialisedValues.Ctx,
-		TestReportPath: testReportPath,
-		Path:           path,
+		TestReportFS:    initialisedValues.TestReportFS,
+		TestReport:      initialisedTestSets.TestReport,
+		Status:          &status,
+		TestSet:         testSet,
+		Success:         &success,
+		Failure:         &failure,
+		Ctx:             initialisedValues.Ctx,
+		TestReportPath:  testReportPath,
+		Path:            path,
+		EnableASNIColor: EnableASNIColor,
 	}
 	status = t.FetchTestResults(resultsCfg)
 	return status
 }
 
-func (t *tester) testHttp(tc models.TestCase, actualResponse *models.HttpResp, noiseConfig models.GlobalNoise, ignoreOrdering bool) (bool, *models.Result) {
+func (t *tester) testHttp(tc models.TestCase, actualResponse *models.HttpResp, noiseConfig models.GlobalNoise, ignoreOrdering bool, EnableASCIColor *bool) (bool, *models.Result) {
```

Comment:
why did the name of the flag suddenly change?

Reply:
Typo, Fixed